VERSION

```diff
@@ -1 +1 @@
-2.3.2
+2.4.0-rc0
```

Comment:
I don't care too much, but so far the version format for release candidates included a dot before the rc number. For example https://github.com/prometheus/prometheus/releases/tag/v2.2.0-rc.1. Both versions are semver compatible, but it might be nice to be consistent.

Reply:
Ah yup, updated it.